frontend/src/components/shared/Button/Button.tsx

```diff
@@ -62,6 +63,10 @@ function Button({
     ComponentType = StyledHeaderButton
   }
 
+  let style = {}
+  if (useContainerWidth === true) {
+    style = { width: "100%" }
```

Comment:
Can you move this CSS to `styled-components.ts`?

Reply:
This is good idea, so the code will be consistent, many thanks 🥇 it's updated 👍